simde/simde-common.h

```diff
@@ -1012,9 +1065,13 @@ HEDLEY_DIAGNOSTIC_POP
        || (HEDLEY_GCC_VERSION_CHECK(9,5,0) && !(HEDLEY_GCC_VERSION_CHECK(10,0,0))))
 #      define SIMDE_BUG_GCC_105339
 #    endif
+#    if defined(SIMDE_ARM_NEON_A32V7_NATIVE) && !defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+#      define SIMDE_BUG_GCC_A32V7_MISSFUNC
+#    endif
 #  elif defined(__clang__)
 #    if defined(SIMDE_ARCH_AARCH64)
 #      define SIMDE_BUG_CLANG_48257
+#      define SIMDE_BUG_CLANG_BAD_VGET_SET_LANE_TYPES
```

Comment:
Please file a bug with llvm and change this to refer to that bug number

Reply:
It has already been filed.
https://github.com/llvm/llvm-project/issues/71362, https://github.com/llvm/llvm-project/issues/71365